modules/scripts/startup-script/templates/startup-script-custom.tpl

```diff
@@ -27,10 +27,12 @@ stdlib::runner() {
 
   stdlib::get_from_bucket -u "gs://${bucket}/$object" -d "$destpath" -f "$filename"
 
+  echo "start executing runner: $filename"
   case "$1" in
     ansible-local) stdlib::run_playbook "$destpath/$filename" "$args";;
     shell) . $destpath/$filename $args;;
   esac
+  echo "finished executing runner: $filename"
   
   exit_code=$?
   stdlib::info "=== $object finished with exit_code=$exit_code ==="
```

Comment:
This line already announces when the runner has completed. I'd stick with this format for the "start executing runner" line as well.

Reply:
What is the difference between using echo here, vs stdlib::info? Shouldn't the latter be preferred?

Reply:
I honestly had never noticed that line in the logs. 

I have:
- removed the finished line
- switched to using stdlib::info
- updated start line format to look similar to finished

We can either merge or close. I think it is marginally better to also print the begining to make it clear the runner has started.

Reply:
It was added recently, with the switch away from source. 
I like including the begin line as well, if a runner stalls you don't have to go back to the blueprint to figure out which one was active currently.